README.md

```diff
@@ -16,6 +16,7 @@ scrapi
     - Install requirements
     - Install Elasticsearch
```

Comment:
Did you add the cp `api/api/settings/local-dist.py api/api/settings/local.py` to the README?